server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -466,7 +466,7 @@ public InputStream readBinaryData(String sessionId, String sqlId, Long rowNum, I
      * Rewrite sqls, will do <br>
      * 1. add ODC_INTERNAL_ROWID query column
      */
-    private List<SqlTuple> generateSqlTuple(List<OffsetString> sqls, ConnectionSession session,
+    public List<SqlTuple> generateSqlTuple(List<OffsetString> sqls, ConnectionSession session,
```

Comment:
if change method scope to public, parameter verification should involved

Reply:
ok

Reply:
Forgot to switch back to `private`